plugins/kosync.koplugin/main.lua

```diff
@@ -25,12 +25,19 @@ local KOSync = InputContainer:new{
     title = _("Register/login to KOReader server"),
 }
 
+-- Creates a unique (hopefully) device id to identify this device. This function will only be
+-- called during the very first time KOSync has been initialized.
+local function createDeviceId()
+    return tostring(math.random(100,999))
```

Comment:
Yes, please use UUID instead. The id should have more entropy and encoding random number with just ascii number is not as efficient as UUID.